From e628f96df051dae5f2d5b6baf89deb9470207478 Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Tue, 13 Jun 2023 05:56:58 +0000
Subject: [PATCH 2/2] add condition for dh_installdocs

This should only be installed when linux_user is enabled.

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 debian/rules | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/debian/rules b/debian/rules
index 67549c4d..2d48c723 100755
--- a/debian/rules
+++ b/debian/rules
@@ -357,10 +357,14 @@ build-arch: $(addprefix build-, ${qemu-builds})
 override_dh_auto_install-arch: $(addprefix install-, ${qemu-builds})
 
 override_dh_installdocs:
+ifeq ($(enable_linux_user),enable)
 	dh_installdocs -Nqemu-user-binfmt
 	dh_installdocs -pqemu-user-binfmt --link-doc=qemu-user
+endif
 override_dh_installchangelogs:
+ifeq ($(enable_linux_user),enable)
 	dh_installchangelogs -Nqemu-user-binfmt
+endif
 override_dh_installinit:
 	dh_installinit -pqemu-guest-agent
 override_dh_installsystemd:
-- 
2.30.2

